internal/driver/checkstatuses.go

```diff
@@ -29,24 +30,29 @@ func (d *Driver) checkStatuses() {
 		go func() {
 			defer wg.Done()
 
+			// if device is unknown, and missing a MAC Address, try and determine the MAC address via the endpoint reference
+			if strings.HasPrefix(device.Name, "unknown_unknown_") && device.Protocols[OnvifProtocol][MACAddress] == "" {
+				if endpointRefAddr := device.Protocols[OnvifProtocol][EndpointRefAddress]; endpointRefAddr != "" {
+					if mac := d.macAddressMapper.MatchEndpointRefAddressToMAC(endpointRefAddr); mac != "" {
+						// we were able to determine the mac address for the device. we set it here which will allow the
```

Comment:
I think this comment needs to be updated instead of using `we`, we can keep it generic as other comments.